IFC4x3/Sections/Domain specific data schemas/Schemas/IfcRailDomain/PropertySets/Pset_RailwayCableCarrier/DocPropertySet.xml

```diff
@@ -0,0 +1,7 @@
+<?xml version="1.0" encoding="utf-8"?>
+<DocPropertySet xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" Name="Pset_RailwayCableCarrier" UniqueId="4a5a4b8b-08db-6a46-9e01-58dfa569763b" ApplicableType="IfcCableCarrierSegment" PropertySetType="PSET_TYPEDRIVENOVERRIDE">
```

Comment:
Is this a type or instance based property set?

Reply:
It's a type based property, I modified the property and add " CableCarrierSegmentType..." commit : bda242612f4cd501c539b8f4b244ab8da0569a6d